sale_packaging_price/models/product.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl
+
+from openerp import api, fields, models, _
+from openerp.addons.decimal_precision import decimal_precision as dp
+
+
+class ProductPackagingMaterial(models.Model):
```

Comment:
It seems the main model here is product.packaging, so I will apply guidelines and rename to `product_packaging.py`, but it seems not splitting is justified this time.